### PR TITLE
feature(cbi-740-ddc-metrics): Rename to meaningfull ocw

### PR DESCRIPTION
**Pull request by huythanhcere**
Draft PR. Buildable, runnable. Dependencies: Call smart contract submit.

### CODE REVIEW

**Comment by evgeny-s** [comment]
You have conflict in one file. Please pull from `feature/ddc-ocw` branch and update it.

**Comment by huythanhcere** [comment]
> You have conflict in one file. Please pull from `feature/ddc-ocw` branch and update it.
I have just pulled feature/ddc-ocw. There is not any conflict. After rebuild successfully, i will notice you.

**Comment by evgeny-s** [comment]
Screenshot from 2021-05-14 11-01-35
Can you see this description below?
This branch has conflicts that must be resolved
?
@huythanhcere

**Comment by huythanhcere** [comment]
So strange: This folder is renamed. frame/example-offchain-worker/src/lib.rs --> frame/ddc-metrics-offchain-worker/src/lib.rs. In local side there is not any conflict.

**Comment by evgeny-s** [comment]
* Pull locally from `feature/ddc-ocw` 
* Merge `feature/ddc-ocw` to your branch and you will face with the conflict
@huythanhcere

**Comment by huythanhcere** [comment]
Out of date base branch. I commit the latest. I did not need to resolve any thing. Now it is ok.

File: bin/node/runtime/src/lib.rs [hunk 1]
```diff
@@ -1008,7 +1008,7 @@ construct_runtime!(
 		ChainBridge: pallet_chainbridge::{Module, Call, Storage, Event<T>},
 		Erc721: pallet_erc721::{Module, Call, Storage, Event<T>},
 		Erc20: pallet_erc20::{Module, Call, Event<T>},
-		ExampleOffchainWorker: pallet_example_offchain_worker::{Module, Call, Event<T>},
+		ExampleOffchainWorker: pallet_ddc_metrics_offchain_worker::{Module, Call, Event<T>},
```
Comment (naure) [inline bin/node/runtime/src/lib.rs:1011]: Rename.

File: frame/ddc-metrics-offchain-worker/src/lib.rs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Offchain worker for DDC metrics.
 //
-// Inspired from https://github.com/paritytech/substrate/tree/master/frame/example-offchain-worker
+// Inspired from https://github.com/paritytech/substrate/tree/master/frame/ddc-metrics-offchain-worker
```
Comment (naure) [inline frame/ddc-metrics-offchain-worker/src/lib.rs:3]: Incorrect renaming.

File: frame/ddc-metrics-offchain-worker/src/lib.rs [hunk 3]
```diff
@@ -230,7 +230,7 @@ impl<T: Trait> Module<T> {
     }
 
     fn check_if_should_proceed(block_number: T::BlockNumber) -> bool {
-        let s_next_at = StorageValueRef::persistent(b"example-offchain-worker::next-at"); // TODO: Rename after OCW renamed
+        let s_next_at = StorageValueRef::persistent(b"ddc-metrics-offchain-worker::next-at"); // TODO: Rename after OCW renamed
```
Comment (naure) [inline frame/ddc-metrics-offchain-worker/src/lib.rs:233]: Remove TODO.

File: frame/system/src/offchain.rs [hunk 4]
```diff
@@ -26,7 +26,7 @@
 //!
 //! ## Usage
 //!
-//! Please refer to [`example-offchain-worker`](../../pallet_example_offchain_worker/index.html) for
+//! Please refer to [`ddc-metrics-offchain-worker`](../../pallet_ddc_metrics_offchain_worker/index.html) for
```
Comment (naure) [inline frame/system/src/offchain.rs:29]: Incorrect renaming.

File: frame/system/src/offchain.rs [hunk 5]
```diff
@@ -46,7 +46,7 @@
 //! #### Submit an unsigned transaction with a signed payload
 //!
 //! Initially, a payload instance that implements the `SignedPayload` trait should be defined.
-//! See [`PricePayload`](../../pallet_example_offchain_worker/struct.PricePayload.html)
+//! See [`PricePayload`](../../pallet_ddc_metrics_offchain_worker/struct.PricePayload.html)
```
Comment (naure) [inline frame/system/src/offchain.rs:49]: Incorrect renaming.

**Review by naure — COMMENTED** [review]
Missing some names in in frame/ddc-metrics-offchain-worker/src/lib.rs:
- ExampleOffchainWorker
- Rename TestAuthId to DdcMetricsAuthId 

- See other comments in PR

File: bin/node/runtime/src/lib.rs (hunk 1 above)
Comment (huythanhcere) [reply bin/node/runtime/src/lib.rs:1011]: Fixed.

File: frame/ddc-metrics-offchain-worker/src/lib.rs (hunk 2 above)
Comment (huythanhcere) [reply frame/ddc-metrics-offchain-worker/src/lib.rs:3]: Fixed

File: frame/ddc-metrics-offchain-worker/src/lib.rs (hunk 3 above)
Comment (huythanhcere) [reply frame/ddc-metrics-offchain-worker/src/lib.rs:233]: Fixed

File: frame/system/src/offchain.rs (hunk 4 above)
Comment (huythanhcere) [reply frame/system/src/offchain.rs:29]: Fixed

File: frame/system/src/offchain.rs (hunk 5 above)
Comment (huythanhcere) [reply frame/system/src/offchain.rs:49]: Fixed

**Pull request merged**